apps/src/blockly/addons/cdoFieldDropdown.ts

```diff
@@ -180,23 +202,26 @@ export default class CdoFieldDropdown extends GoogleBlockly.FieldDropdown {
     /**
      * Begin CDO customization
      */
-    this.arrow.setAttribute('dominant-baseline', 'central');
+    arrow.setAttribute('dominant-baseline', 'central');
     /**
      * End CDO customization
      */
 
     if (this.getSourceBlock()?.RTL) {
-      this.getTextElement().insertBefore(this.arrow, this.textContent_);
+      this.getTextElement().insertBefore(arrow, this.textContent_);
     } else {
-      this.getTextElement().appendChild(this.arrow);
+      this.getTextElement().appendChild(arrow);
     }
+    // this.arrow is private in the parent.
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    (this as any).arrow = arrow;
```

Comment:
I just created an issue to help us track when we might be able to remove this override: https://github.com/google/blockly/issues/7890

Reply:
thanks! I'll add a comment with that link